### PR TITLE
Added support for PDF Rotation

### PR DESCRIPTION
**Pull request by theonlysake**
resolves: #126

### CODE REVIEW

File: example/src/Sidebar.tsx [hunk 1]
```diff
@@ -11,6 +11,10 @@ const updateHash = (highlight: IHighlight) => {
   document.location.hash = `highlight-${highlight.id}`;
 };
 
+const setPagesRotation = (rotation: Number) => {
+  window.PdfViewer.viewer.pagesRotation = rotation;
```
Comment (holub008) [inline example/src/Sidebar.tsx:15]: Currently, this variable _may_ not be considered official API. The [location setting it](https://github.com/agentcooper/react-pdf-highlighter/blob/1d1c7684c5842104a2d01c9f81c992557dfadaa7/src/components/PdfHighlighter.tsx#L193) calls it debug, so I'm not sure we should expect clients to interact with it. (Darian, I know we use it in our app for searching, but we're probably being naughty)

Perhaps a cleaner API (doesn't treat the viewer as a global, better abstracted):

- provide a `rotationDegrees` prop to `PDFHighlighter`
    - default `0` for backwards compatibility
- inside `PDFHighlighter`, perform a `componentDidUpdate` check on `rotationDegrees` which updates `this.viewer`

**Comment by holub008** [comment]
@agentcooper For transparency, @theonlysake and I are coworkers; we hope you find this a worthwhile improvement!

File: example/src/Sidebar.tsx (hunk 1 above)
Comment (theonlysake) [reply example/src/Sidebar.tsx]: done! I named the prop `pagesRotation` for transparency reasons

Comment (holub008) [reply example/src/Sidebar.tsx]: got it- that seems like a somewhat lousy name from PDF.js, but I can see wanting to keep it consistent

**Pull request closed without merging**